Sources/HermesAppDelegate.m

```diff
@@ -69,13 +69,9 @@ - (NSMenu *)applicationDockMenu:(NSApplication *)sender {
   NSMenuItem *menuItem;
   Song *song = [[playback playing] playingSong];
   if (song != nil) {
-    [menu addItemWithTitle:@"Now Playing" action:nil keyEquivalent:@""];;
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song title]]
-                    action:nil
-             keyEquivalent:@""];
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song artist]]
-                    action:nil
-             keyEquivalent:@""];
+    [menu addItemWithTitle:nowPlaying.title action:nil keyEquivalent:@""];;
+    [menu addItemWithTitle:song.title action:nil keyEquivalent:@""];
+    [menu addItemWithTitle:song.artist action:nil keyEquivalent:@""];
     [menu addItem:[NSMenuItem separatorItem]];
```

Comment:
So I guess I was just trying to imitate what was happening with the status bar icon.  Either way works for me, just let me know.